sajibportfolio.css

```diff
@@ -0,0 +1,901 @@
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  background-color: #dfe1e6;
+  margin-bottom: 2px;
+  border-radius: 10px;
+  align-items: center;
+}
+
+#spec_name {
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  order: 0;
+  flex-grow: 0;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.lfs {
+  display: flex;
+  flex-direction: row;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  font-weight: 600;
+}
+
+.lhr {
+  display: flex;
+  flex-direction: row;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.lfsi {
+  display: flex;
+  flex-direction: row;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.img1 {
+  display: flex;
+  flex-direction: row;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding: 0 15px;
+  height: 30px;
+}
+
+.lng1 {
+  display: flex;
+  flex-direction: row;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.llt1 {
+  position: absolute;
+  height: 24px;
+  left: 76px;
+  right: 12px;
+  top: calc(50% - 24px / 2 - 0.36px);
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: #253858;
+}
+
+.lli1 {
+  position: absolute;
+  width: 48px;
+  height: 48px;
+  left: 40px;
+  top: calc(50% - 48px / 2 - 0.36px);
+  background: url(statics/images/java.png);
+  border: 1px solid #fff;
+}
+
+.nbs1 {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 26px;
+  top: calc(50.5%);
+  border-radius: 8px;
+}
+
+#presentation {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+p:hover {
+  text-decoration: underline;
+  text-decoration-color: #396df2;
+  text-decoration-thickness: 2px;
+}
+
+#profile_link {
+  display: flex;
+  flex-direction: column;
+}
+
+#profile_link h2 {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+#profile_link h2:hover {
+  text-decoration: underline;
+  text-decoration-color: #18a0fa;
+  text-decoration-thickness: 3px;
+}
+
+#profile_link ul {
+  display: flex;
+  padding-left: 0;
+}
+
+#profile_link ul li {
+  list-style: none;
+  margin-right: 12px;
+}
+
+#my_resume {
+  text-decoration: none;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 5px;
+  text-align: center;
+  padding: 10px 0;
+  align-self: baseline;
+}
+
+#my_resume:hover {
+  text-decoration: underline;
+  text-decoration-color: black;
+  text-decoration-thickness: 1px;
+}
+
+#my_skills {
+  display: flex;
+  flex-direction: column;
+  margin-top: 35px;
+  margin-left: 24px;
+}
+
+#languages_skills {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- [ ] As stated the project requirement You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly adjust the alignment of "About me" section that was in introduced to address the suggested changes , Please Fix it before the merge to main branch.

Your About me section at the minimum mobile screen size of 375px
![Capture](https://user-images.githubusercontent.com/93598189/205457260-8622fc7c-1717-4eed-a10a-51a3a83793f0.PNG)

Reply:
![image](https://user-images.githubusercontent.com/39811497/205457717-6960d046-978b-4c9a-af4b-766de5a908b4.png)

solved

Reply:
Hello @sajibarafatsiddiqui Great 👍